core/cmake/smeConfig.cmake.in

```diff
@@ -12,16 +12,6 @@ find_dependency(cereal REQUIRED)
 find_dependency(Pagmo REQUIRED)
 find_dependency(TBB CONFIG REQUIRED)
 find_dependency(combine-static REQUIRED CONFIGS Combine-static-config.cmake)
-# provide METIS::METIS target expected by dune-copasi
-add_library(
-  METIS::METIS
-  INTERFACE
-  IMPORTED)
-find_dependency(SCOTCH REQUIRED)
-target_link_libraries(
-  METIS::METIS
-  INTERFACE SCOTCH::scotchmetisv3
-  INTERFACE SCOTCH::scotcherr)
 find_dependency(dune-common REQUIRED)
 find_dependency(dune-geometry REQUIRED)
 find_dependency(dune-uggrid REQUIRED)
```

Comment:
Shouldn't those also be removed?

Reply:
yes, good catch! by the way the tests are failing for me now in CI when I build dune-copasi: https://github.com/spatial-model-editor/sme_deps/actions/runs/7737953815